Signal/src/ViewControllers/OWSTableViewController.h

```diff
@@ -64,6 +64,10 @@ typedef UITableViewCell *_Nonnull (^OWSTableCustomCellBlock)();
 + (OWSTableItem *)itemWithCustomCellBlock:(OWSTableCustomCellBlock)customCellBlock
                               actionBlock:(nullable OWSTableActionBlock)actionBlock;
 
++ (OWSTableItem *)disclosureItemWithText:(NSString *)text actionBlock:(nullable OWSTableActionBlock)actionBlock;
+
++ (OWSTableItem *)actionItemWithText:(NSString *)text actionBlock:(nullable OWSTableActionBlock)actionBlock;
```

Comment:
These factory methods will help us DRY up a ton of code & maintain consistent styles across our table views.